onnxscript/ir/_name_authority.py

```diff
@@ -12,20 +12,53 @@ class NameAuthority:
     ``node_{op_type}_{node_counter}`` for nodes. The counter is incremented each time
     a new value or node is named.
 
-    The class does not keep track of the names it has given, so it is possible to
-    generate names that conflicts with existing names. It is the responsibility of the
-    user to ensure that the names are unique (typically by running a name-fixing pass
-    on the graph).
+    This class does keep track of the names it has given and all the existing names
```

Comment:
nit rewording suggestion: `This class keeps tracks of the names it has generated and existing names`

Reply:
Done